bin/load-env.sh

```diff
@@ -11,10 +11,18 @@ then
   exit 1
 fi
 
-if [ ! -f "$FLUO_PROPS" ]
-then
+if [ -z "$APP_NAME" ]; then
+  echo "APP_NAME is not set!" 
+  exit 1
+fi
+
+if [ ! -d $FLUO_HOME/apps/$APP_NAME ]; then
+  $FLUO_HOME/bin/fluo new $APP_NAME
```

Comment:
this is an odd place to call new... I was looking for the call in the configure script....